app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,11 @@
+class LikesController < ApplicationController
+  def create
+    @post = Post.find(params[:post_id])
+    @like = @post.likes.new(user: current_user)
+    if @like.save
+      redirect_to user_post_path(@post.user, @post), notice: 'Post liked successfully.'
+    else
```

Comment:
- [ ] I was able to like a post successfully but I still got the error message below 👇 for smooth running of your app. ℹ️ 

<img width="887" alt="Untitled" src="https://github.com/Young-Gaius/Rails-Blog-App/assets/103386183/7635bd19-012b-42eb-9206-f32c78ad9741">